requirements.txt

```diff
@@ -104,7 +104,7 @@ sticht==1.1.18
 strict-rfc3339==0.7
 swagger-spec-validator==2.1.0
 syslogmp==0.2.2
-task-processing==0.1.3
+task-processing==0.12.1
```

Comment:
do we actually use task_processing in paasta right now? 😮

Reply:
It's "used" in the legacy `remote run` implementation